lib/puppet/provider/ec2_vpc/v2.rb

```diff
@@ -0,0 +1,94 @@
+require_relative '../../../puppet_x/puppetlabs/aws.rb'
+
+Puppet::Type.type(:ec2_vpc).provide(:v2, :parent => PuppetX::Puppetlabs::Aws) do
+  confine feature: :aws
+
+  mk_resource_methods
+  remove_method :tags=
```

Comment:
If you run the code you should find you can modify tags.

It's there because `mk_resource_methods` creates a `tags=` method on the new class. But the parent class has the implementation.

`remove_method`: "Removes the method identified by symbol from the current class. For an example, see Module.undef_method."

So this removes the method added by `mk_resource_methods` and uses the one in the base class instead.